addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -37,10 +37,20 @@
     website.ready().done(function() {
 
         // Store social share data to display modal on next page
-        $(document.body).on('click', '.oe_social_share_call', function() {
-            var social_data = {};
-            social_data['target_type'] = $(this).data('social-target-type');
-            sessionStorage.setItem('social_share', JSON.stringify(social_data));
+        $('.oe_social_share_call').parent().submit(function(event) {
```

Comment:
@tde-banana-odoo yeah! it's work if there is class "karma_required", its good. but if there is error at python side and Traceback(mean controller not return success) then 'Thanks for posting' message will occurred when page is refreshed.